using-valtimo/plugin/documenten-api/configure-plugin.md

```diff
@@ -0,0 +1,53 @@
+# Using the Documenten API Plugin
+
+The Documenten API plugin can be used to store documents and their metadata in an application that supports the
+Documenten API standard created and used by the Dutch government.
+
+## Configure the plugin
+
+In order to use this plugin a configuration has to be created. A general description on how to configure
+plugins can be found [here](../configure-plugin.md).
+
+If the Documenten API plugin is not visible in the plugin menu, it is possible the application is missing a dependency.
+Instructions on how to add the OpenZaak dependency can be found
```

Comment:
You mention the OpenZaak dependency while you talk about the visibility of the Documenten API plugin